ivy/functional/ivy/manipulation.py

```diff
@@ -376,12 +376,12 @@ def reshape(
 @handle_out_argument
 @handle_nestable
 def roll(
-    x: Union[ivy.Array, ivy.NativeArray],
+    x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
     shift: Union[int, Tuple[int, ...]],
     axis: Optional[Union[int, Tuple[int, ...]]] = None,
```

Comment:
We should use `Sequence[int]` always, instead of `Tuple[int]` or `Tuple[int, ...]` etc. because Sequence encompasses the use of tuples or lists. So we are extending the standard here.
 
Also, make these changes in backend specific files as well. Thankyou!